text/0097-log-data-model.md

```diff
@@ -0,0 +1,971 @@
+[Appendix B](#appendix-b-severitynumber-example-mappings) for more mapping
+examples.
+
+#### Displaying Severity 
+
+The following table defines the recommended short name for each
+`SeverityNumber` value. The short name can be used for example for representing
+the `SeverityNumber` in the UI:
+
+SeverityNumber|Short Name
+--------------|----------
+1             |TRACE
+2             |TRACE2
+3             |TRACE3
+4             |TRACE4
+5             |DEBUG
+6             |DEBUG2
+7             |DEBUG3
+8             |DEBUG4
+9             |INFO
+10            |INFO2
+11            |INFO3
+12            |INFO4
+13            |WARN
+14            |WARN2
+15            |WARN3
+16            |WARN4
+17            |ERROR
+18            |ERROR2
+19            |ERROR3
+20            |ERROR4
+21            |FATAL
+22            |FATAL2
+23            |FATAL3
+24            |FATAL4
+
+When an individual log record is displayed it is recommended to show both
+`SeverityText` and `SeverityNumber` values. A recommended combined string in
+this case begins with the short name followed by `SeverityText` in parenthesis.
+
+For example "Informational" Syslog record will be displayed as **INFO
+(Informational)**. When for a particular log record the `SeverityNumber` is
+defined but the `SeverityText` is missing it is recommended to only show the
+short name, e.g. **INFO**.
+
+When drop down lists (or other UI elements that are intended to represent the
+possible set of values) are used for representing the severity it is preferable
+to display the short name in such UI elements.
+
+For example a dropdown list of severities that allows filtering log records by
+severities is likely to be more usable if it contains the short names of
+`SeverityNumber` (and thus has a limited upper bound of elements) compared to a
+dropdown list, which lists all distinct `SeverityText` values that are known to
+the system (which can be a large number of elements, often differing only in
+capitalization or abbreviated, e.g. "Info" vs "Information").
+
+#### Comparing Severity
+
+In the contexts where severity participates less-than / greater-than comparisons
+`SeverityNumber` field should be used. `SeverityNumber` can be compared to
+another `SeverityNumber` or to numbers in the 1..24 range (or to the
+corresponding short names).
+
+When severity is used in equality or inequality comparisons (for example in
+filters in the UIs) the recommendation is to attempt to use both `SeverityText`
+and short name of `SeverityNumber` to perform matches. For example if we have a
+record with `SeverityText` field equal to "Informational" and `SeverityNumber`
+field equal to INFO then it may be preferable from the user experience
+perspective to ensure that **severity="Informational"** and **severity="INFO"**
+conditions both to are TRUE for that record.
+
+### Field: `ShortName`
+
+Type: string.
+
+Description: Short event identifier that does not contain varying parts.
```

Comment:
Namespace might be required?

Reply:
I am not sure about it. There is no uniqueness requirement for this field so I don't know how the namespace would be used and for what purpose.

Reply:
Resolving. Feel free to reopen if needed.